.env.sample

```diff
@@ -3,15 +3,17 @@
 # Rename this file to .env and fill in the values accordingly.
 # Xseed
 ## Download Client Names
-TORRENT_CLIENT_NAME="" # Example: "Qbit"
-USENET_CLIENT_NAME=""  # Example: "SABnzbd"
+### For multiple clients, use format: client1,client2,client3
+TORRENT_CLIENTS="" # Examples: "Qbit", "Qbit,Deluge"
+USENET_CLIENTS=""  # Examples: "SABnzbd", "SABnzbd,SABnzbd Anime"
 ## Cross Seed API configuration
 XSEED_HOST="" # Example: "crossseed"
 XSEED_PORT="" # Example: "2468"
 ## API Key for Cross Seed, if applicable
 XSEED_APIKEY="" # Example: "your-api-key"
 ## Path to store the script's database of prior searches
 LOG_FILE="" # Example: "/config/xseed_db.log"
+LOGID_FILE="" # Example: "/config/xseed-id.log"
```

Comment:
This is not relevant to this PR and needs to be a seperate commit and a seperate PR

Reply:
#28